COPYING-BSD

```diff
@@ -1,4 +1,4 @@
-Copyright (C) 2012-2021, Blue Brain Project/EPFL.
+Copyright (c) 2016-2022 Blue Brain Project/EPFL
```

Comment:
Is it expected to remove the end dot and to set `(C)` to lower case?

Reply:
the end dot is could be there, but we settled on a small "c".

Reply:
Ok, thanks.
The actual choice is not important to me but I just wanted to be sure that it was changed on purpose. So I guess we can merge this.